source/circle.html.erb

```diff
@@ -29,22 +29,30 @@ title: Circle Membership
 
 <div class="main">
   <div class="main-left">
+  <div class="circle-members-header">
+    <img class="circle-logo" src="../images/circle-member-half-logo.png">
```

Comment:
Should the other logo image be removed since it looks like we're using this one now?